experiment.py

```diff
@@ -142,11 +186,9 @@ def Scan_Field(stop):
         client.field.approach_mode.linear,
         client.field.driven_mode.driven
     )
-
     client.set_temperature(
         300,
-        2,
+        3,
         client.temperature.approach_mode.fast_settle
     )
-
-    
\ No newline at end of file
+    time.sleep(100 * 60)

```

Comment:
We should use client.wait_for here instead of this time.sleep. It should have the same effect.